pkgs/development/python-modules/pytest-mpi/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, sybil
+, pytest
+, pytestCheckHook
+, mpi
+, mpi4py
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-mpi";
+  version = "0.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "aragilar";
+    repo = "pytest-mpi";
+    rev = "v${version}";
+    hash = "sha256-m3HTGLoPnYeg0oeIA1nzTzch7FtkuXTYpox4rRgo5MU=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+  dependencies = [
+    pytest
```

Comment:
We don't propagate pytest to make overrides to it easier. Please move it to buildInputs

Reply:
> We don't propagate pytest to make overrides to it easier. Please move it to buildInputs

OK good idea, thanks :).